tests/phpunit/compat/class-wc-stripe-subscription-renewal-test.php

```
@@ -257,6 +257,219 @@ public function test_renewal_successful() {
+		WC_Stripe_Helper::update_main_stripe_settings( $stripe_settings );
+
+		$logged_warnings = [];
+		$logger          = $this->getMockBuilder( WC_Logger::class )->disableOriginalConstructor()->getMock();
+		$logger->method( 'warning' )->willReturnCallback(
+			function ( $message ) use ( &$logged_warnings ) {
+				$logged_warnings[] = $message;
+			}
+		);
+		WC_Stripe_Logger::$logger = $logger;
+
+		$pre_http_request_response_callback = function ( $preempt, $request_args, $url ) use ( $expected_api_endpoint, &$api_requests ) {
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $pre_http_request_response_callback. Keep variable name length under 20. (undefined)

(LongVariable)